integration-requirements.txt

```diff
@@ -1,4 +1,6 @@
-pytest-asyncio==0.21.1
+# TODO: remove when new version is released
+# Relevant PR https://github.com/go-macaroon-bakery/py-macaroon-bakery/pull/95
+macaroonbakery==1.3.2
```

Comment:
why does this charm need the macaroon bakery?

Reply:
There was a bug with protobuffs that broke the CI, I think the dependency comes from pythonlib-juju